internal/imagedefinition/image_definition.go

```diff
@@ -97,7 +97,7 @@ type PPA struct {
 	PPAName     string `yaml:"name"         json:"PPAName"               jsonschema:"pattern=^[a-zA-Z0-9_.+-]+/[a-zA-Z0-9_.+-]+$"`
 	Auth        string `yaml:"auth"         json:"Auth,omitempty"        jsonschema:"pattern=^[a-zA-Z0-9_.+-]+:[a-zA-Z0-9]+$"`
 	Fingerprint string `yaml:"fingerprint"  json:"Fingerprint,omitempty"`
-	KeepEnabled bool   `yaml:"keep-enabled" json:"KeepEnabled"           default:"true"`
+	KeepEnabled *bool  `yaml:"keep-enabled" json:"KeepEnabled"           default:"true"`
```

Comment:
As for this, I'll mention more about it in my main review comment, but I missed that it's not used! I was sure we used it already!